### PR TITLE
MYKIInc.MYKIDesktopApp version 1.4.4

### PR DESCRIPTION
**Pull request by cdwalters**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16504)

### CODE REVIEW

File: manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: MYKIInc.MYKIDesktopApp
```
Comment (ItzLevvie) [inline manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.yaml:3]:
```suggestion
PackageIdentifier: MYKI.MYKI
```

File: manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MYKIInc.MYKIDesktopApp
+PackageVersion: 1.4.4
+PackageLocale: en-US
+Publisher: MYKI Inc.
+PackageName: MYKI Desktop App
```
Comment (ItzLevvie) [inline manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.locale.en-US.yaml:7]:
```suggestion
PackageName: MYKI
```
image

File: manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MYKIInc.MYKIDesktopApp
```
Comment (ItzLevvie) [inline manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: MYKI.MYKI
```

File: manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MYKIInc.MYKIDesktopApp
```
Comment (ItzLevvie) [inline manifests/m/MYKIInc/MYKIDesktopApp/1.4.4/MYKIInc.MYKIDesktopApp.installer.yaml:3]:
```suggestion
PackageIdentifier: MYKI.MYKI
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image


Just make sure to update all of the folders and file names using the GitHub Desktop client on Windows.

**Comment by cdwalters** [comment]
Is there anything that needs to be done to be submitted?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Since you're reading this from email, open up https://github.com/microsoft/winget-pkgs/pull/16504 and click on a button that says "Commit suggestion" or "Add suggestions to batch."

**You'll also need to have the GitHub Desktop client on Windows to update the folders and the file names**.

**Comment by cdwalters** [comment]
I have no idea what you want me to do.

**Comment by ItzLevvie** [comment]
> I have no idea what you want me to do.

1) Download the client from https://desktop.github.com
2) Login with your details
3) Clone your fork of winget-pkgs
4) Choose the `autogenerated/MYKIInc.MYKIDesktopApp/2d4421e0-39ca-4db3-99c8-738afdf7a070` branch by clicking on the `Current branch` button.
5) Open up File Explorer and change the folder and file names to match `MYKI.MYKI`, e.g. change `MYKIInc` to `MYKI` and change `MYKIDesktopApp` to `MYKI`.
6) Go back to the client and write a commit message and click on the blue button at the bottom.
7) Push your changes to the branch by clicking on the `Push origin` button.

**Comment by cdwalters** [comment]
It won't let me do anything.

**Pull request closed without merging**